src/huggingface_hub/_shtab.py

```diff
@@ -0,0 +1,39 @@
+"""Fake shtab."""
```

Comment:
Could you move this file to `huggingface_hub/commands/_shtab.py` please ?

Reply:
OK.